function otraFuncion(){
     serompe();
}

function serompe(){
    return 3 + z;
}

function asyncFunction(){
    setTimeout(() => {
       try{
           return 3 + z;
       }catch(err){
        console.error('Ocurrio un error en la funciÃ³n async ->'+err.message);
       }
    }, 1000);
}



// IMPRESIÃ“N
try {
    asyncFunction();
} catch(err){
    console.error('Vaya, algo se ha roto ... -> '+err.message);
}